File: auth-service/src/main/java/com/jaksimsam1/authservice/domain/auth/repository/jpa/AuthJPARepository.java
package com.jaksimsam1.authservice.domain.auth.repository.jpa;

import com.jaksimsam1.authservice.domain.auth.entity.Auth;
import org.springframework.data.jpa.repository.JpaRepository;

import java.util.UUID;

public interface AuthJPARepository extends JpaRepository<Auth, UUID> {
    Auth findByEmail(String email);
    Auth findByEmailAndPassword(String email, String password);
    void deleteByUserId(UUID userId);
}

File: auth-service/src/main/kotlin/com/jaksimsam1/authservice/domain/auth/repository/jpa/AuthJpaRepository.kt
package com.jaksimsam1.authservice.domain.auth.repository.jpa

import com.jaksimsam1.authservice.domain.auth.entity.AuthUsers
import org.springframework.data.jpa.repository.JpaRepository

interface AuthJpaRepository: JpaRepository<AuthUsers, String> {
    suspend fun findByEmailAndPassword(email: String, password: String): AuthUsers
}